props/saber_fett263_buttons.h

```diff
@@ -5295,7 +5300,7 @@ SaberFett263Buttons() : PropBase() {}
           return true;
         }
 #endif
-        if (!battle_mode_ || swinging_) {
+        if ((!battle_mode_) || (battle_mode_ && swinging_)) {
```

Comment:
This does exactly the same as what you had before, but less clearly.

Reply:
It fixed a "hole", If it's not battle mode this should only do a clash ever, but if it is battle mode and the blade doesn't stop swinging then it should do a clash, the previous would let it through this check outside battle_mode and create a lockup if you had a clash without the swing stopping.  I added a comment to help clarify.

Reply:
Ok, time for logic tables:

B=battle_mode, S=swinging_, T=true, F=false

battle_mode:
__|_S__!S_
 B|   T    T 
!B|   F    F

swinging_:
__|_S__!S_
 B|   T    F 
!B|   T    F

!battle_mode_
__|_S__!S_
 B|   F    F 
!B|   T    T

!battle_mode || swinging_
__|_S__!S_
 B|   T    F 
!B|   T    T

battle_mode && swinging_
__|_S__!S_
 B|   T    F 
!B|   F    F

(!battle_mode) || (battle_mode_ && swinging_)
__|_S__!S_
 B|   T    F 
!B|   T    T

Assuming you get what the logic tables mean, you'll see that 
     (!battle_mode || swinging_)
and
     ((!battle_mode_) || (battle_mode_ && swinging_))
have exactly the same results in all cases, so it's not possible for this change to change any behavior, and it looked better before.

Reply:
Makes sense (or should have I guess), I reverted.